packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -75,10 +75,11 @@ function SingleImageViewer({
           <title id={title.id}>{title.text}</title>
         )}
         <svg
-          style={{ touchAction: 'none' }}
+          style={{ touchAction: 'pinch-zoom' }}
           viewBox={`0 0 ${naturalWidth} ${naturalHeight}`}
         >
           <VisXZoom
+            disablesScrolling={false}
```

Comment:
I made this true by default because the JSON data viewers disable window scrolling.